### ISSUE TITLE
Improve visibility of polls for beatmaps nominated for Loved

### ISSUE THREAD

**Issue by ubinici** [ppy/osu-web#8202]
Currently, a typical user needs to go through newspost -> beatmap page -> download + play (if unfamiliar) -> newspost again -> forum thread in order to vote upon just one of the beatmaps nominated for Loved. Given how layered and tedious to navigate the current system is, most users simply don't bother voting for any of these nominations. 

We've recently discussed ways to alleviate this visibility issue and reduce the layers for the better in the `#osu-loved` channel. As a result, we've come up with the following solutions:

- **Enable users to vote directly on the beatmap page.** This is the primary goal we would like to achieve, albeit the most complex one. 
   - Upon pushing the `Love` button the beatmap will be moved to an "In Voting" state, where the Hype Train section will be replaced with Yes and No buttons for users to vote.
   - Prior to the user casting their vote, there should also be a check on whether the user has played the map before.
   - The polls will be closed either manually by a Project Loved Team member or automatically after 2 weeks. Depending on the results, the beatmap will be moved to Loved or fall back to the Pending state.

- **Add a button/banner redirecting users to the already existing forum polls to the beatmap page.** This will keep the layers intact, but still improve the visibility of the polls to some extent.
   - Just a simple button to be manually adjusted by a Project Loved Team member will probably be enough.   
   - As much as it is easier to implement, I personally believe it creates more burdens on the shoulders of the team members rather than improving their lives. 

There is also #7641 that I believe can further enhance the visibility of Project Loved quite a bit, which can be perceived as an alternative to the aforementioned suggestions.

---

I think I've compiled all the ideas discussed in the channel, but pinging @huuishuu and @cl8n just in case I forgot something.

**Comment by peppy** [comment]
That looks pretty good, although the description area may be a bit wordy.

@arflyte any changes you'd want made to that UI above? It will be on the beatmap discussion page where the hype section currently is.

**Comment by cl8n** [comment]
here's more context for https://github.com/ppy/osu-web/issues/8202#issuecomment-945756100 -- @arflyte would appreciate feedback or required changes to the design so I can move forward with this

the whole panel is basically another view for a loved forum poll ([here's a random example](https://osu.ppy.sh/community/forums/topics/1429931)). these polls always have the same format: Yes/No options, a required threshold of Yes votes for the map to "pass" voting, and a description of the map's content & history written by "captains" aka people who select the maps.

the first screenshot i provided is an example of a map during voting, and second is a map that passed voting. for relevant maps it replaces the WIP/hypetrain panel [on the main beatmap page](https://user-images.githubusercontent.com/9954349/141665972-6293fbbf-db9a-4c0d-af45-ea89bd3f40f5.png). in the screenshots, i have this panel overriding the page hue to the pink one because I thought it matches the loved theme better. if there's anything else that's not self-explanatory in my mockups then I guess that means they need more work :^)

for maps that don't pass voting, I didn't build anything yet, but I think it'd make sense to show the poll results for a week or so before this panel goes away and the map is treated like any other again.

also @peppy I intentionally picked a longer description there, I think most are around a paragraph. can be coordinated with the loved team if future descriptions need to be more strictly edited or something.

**Comment by arflyte** [comment]
Looks good. The only thing I'd change is the threshold bar: I'd use solid green and red lines confined within the bars. Everything else looks good.
